web/src/views/DandisetLandingView/ShareDialog.vue

```diff
@@ -7,7 +7,10 @@
     max-width="500"
   >
     <template #activator="{ on }">
-      <div v-on="on">
+      <div
+        class="d-inline"
+        v-on="on"
+      >
         <v-icon
           color="primary"
```

Comment:
```suggestion
          color="primary"
          style="cursor: pointer;"
```

Reply:
@djarecka this should resolve the last open thread - let me know if you have any questions. I think this PR is good to go after this is applied.